backend/dataInterface/mongoDB.js

```diff
@@ -90,56 +284,63 @@ module.exports.createUser = async (user) => {
     let emailExist = await collection.findOne({ email: user.email });
     if (emailExist) {
       return { Error: "Email already exists" };
-    }
-    else{
+    } else {
       let hashedPassword = await bcrypt.hash(user.password, 10);
       let vettedUser = {
         username: user.username,
         email: user.email,
         password: hashedPassword,
         firstName: user.firstName,
-        lastName: user.lastName
+        lastName: user.lastName,
       };
       const result = await collection.insertOne(vettedUser);
-      return { newObjectId: result.insertedId, message: `User created! ID: ${result.insertedId}`, pass: hashedPassword };
+      return {
+        newObjectId: result.insertedId,
+        message: `User created! ID: ${result.insertedId}`,
+        pass: hashedPassword,
```

Comment:
Returning a password isn't a secure choice. The password should be something that only the user knows.

Reply:
wanted to check if the hash works, it will not be in production